utils/scripts/testconfig/generateTestConfigFile.py

```diff
@@ -92,6 +100,10 @@ def generate_test_config_file(config_template, config_data, test_cluster, set_pr
         prometheus_token = get_prometheus_token(test_cluster, "redhat-ods-monitoring")
         data["RHODS_PROMETHEUS_TOKEN"] = prometheus_token
 
+        # Get prometheus url
```

Comment:
I don't know when set_promotheus_token = false but maybe it should be called set_promotheus_config or similar to imply that it's the token but also the url that are going to be set

```
    # Get prometheus token for test cluster
    if bool(set_promotheus_token):
        prometheus_token = get_prometheus_token(test_cluster, "redhat-ods-monitoring")
        data["RHODS_PROMETHEUS_TOKEN"] = prometheus_token

        # Get prometheus url
        prometheus_url = get_prometheus_url("redhat-ods-monitoring")
        data["RHODS_PROMETHEUS_URL"] = prometheus_url
```

Reply:
Good catch ! Incorporated the comments.